### PR TITLE
added the ability to add a label for radio button and checkbox

### PR DESCRIPTION
**Pull request by DianaRenz**

### CODE REVIEW

File: portal/code/cx-portal-shared-components/src/components/basic/Checkbox/index.tsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-  size?: 'medium' | 'small'
+  size?: 'medium' | 'small',
+  label?: string | number
 }
 
-export const Checkbox = ({ size = 'medium', ...props }: CheckboxProps) => {
-  return <MuiCheckbox size={size} {...props} {...label} />
+export const Checkbox = ({ size = 'medium', label, ...props }: CheckboxProps) => {
+  return label 
+  ? <FormControlLabel disabled control={<MuiCheckbox size={size} {...props} {...ariaLabel} />} label={label} />
```
Comment (oyo) [inline portal/code/cx-portal-shared-components/src/components/basic/Checkbox/index.tsx]: in Storybook the checkbox seems to be disabled permanently - maybe from this setting?

**Review by oyo — CHANGES_REQUESTED** [review]
see the disabled checkbox comment

File: portal/code/cx-portal-shared-components/src/components/basic/Checkbox/index.tsx (hunk 1 above)
Comment (DianaRenz) [reply portal/code/cx-portal-shared-components/src/components/basic/Checkbox/index.tsx]: Yeeees! Thank you Martin :) 
It's removed now

Comment (oyo) [reply portal/code/cx-portal-shared-components/src/components/basic/Checkbox/index.tsx]: thanks - working now ;)

**Review by oyo — APPROVED** [review]

**Review by christianraysono — APPROVED** [review]

**Pull request merged**